pkg/controller/manager.go

```diff
@@ -52,6 +52,20 @@ func (m *Manager) UpdateController(name string, params ControllerParams) {
 	m.updateController(name, params)
 }
 
+func (m *Manager) delayUpdate(ctrl *managedController, params ControllerParams) {
+	select {
+	case <-time.After(params.DelayUpdate):
+		m.mutex.Lock()
+		select { // Refer to func (m *Manager) createControllerLocked
+		case ctrl.update <- params:
+		default:
+		}
```

Comment:
Could this lose updates if `runController()` is not ready to receive the update event?

Maybe this is more robust?
```suggestion
		// Attempt to send the parameters update unless there is a race to shut down.  
		select {
		case ctrl.update <- params:
		case <-ctrl.stop:
		}
```